build/README_zh.md

```diff
@@ -0,0 +1,81 @@
+# build目录说明

```

Comment:
I have the feeling that we should convert the README_zh.md to README.md. Otherwise, people will not easily find this page.

Reply:
hhh, I think only GFW has this problem, so I just wrote this suggestion in this document(README_**zh**.md).